src/lib/filter-style-keys.ts

```diff
@@ -1,15 +1,17 @@
-'use strict';
+export default function filterStyleKeys(
+  styleObject,
+  propertyList?,
+  isAllowlist?
+): Record<string, any> {
```

Comment:
would it be safe to assume this is a `Record<string, string>` ? Or is this some weird `CSSRule` type?

Reply:
Yeah, that is what I ended up doing: https://github.com/braintree/inject-stylesheet/pull/10/files#diff-ef5ce68302bdcca7d05c7cb7f6d6cf3aR17